.github/actions/setup-pnpm-with-dependencies/action.yaml

```
@@ -71,3 +71,26 @@ runs:
       with:
         path: './node_modules'
         key: ${{ steps.node-version.outputs.cache-key }}
+
+    - name: Get Playwright Chromium version
+      id: get-playwright-version
+      run: echo "PLAYWRIGHT_VERSION=$(jq -r '.devDependencies["playwright-chromium"]' < package.json | sed 's/[^0-9.]//g' | sed 's/\./-/g')" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - name: Set Playwright path
+      id: playwright-path
+      shell: bash
+      run: echo "PLAYWRIGHT_BROWSERS_PATH=/home/runner/.cache/ms-playwright" >> $GITHUB_OUTPUT
```

Two issues:

- I believe you actually need to expose this variable to the `$GITHUB_ENV` and then pass this env variable to the step that runs the install command so that Playwright will utilize it during the install 
- I believe the path should be this:

```suggestion
      run: echo "PLAYWRIGHT_BROWSERS_PATH=/home/ubuntu/.cache/ms-playwright" >> $GITHUB_ENV
```

---

Done, and I also replaced `/home` to `$HOME`